sequelizeDatabase/sequelFunctions.js

```diff
@@ -259,23 +259,15 @@ deleteIndividual = IndividualID => {
   //
   // Modfying this to avoid a race condition -- Aaron
   return new Promise((resolve, reject) => {
-    models.EmployeeHistory.destroy({
-      where: {
-        IndividualID: IndividualID
-      }
-    })
-      .then(deletedHistory => {
-        models.Individuals.destroy({
-          where: {
-            IndividualID: IndividualID
-          }
-        }).then(deletedIndividual => {
-          console.log("deletion resolved");
-          resolve(deletedIndividual);
-        });
-      })
-      .catch(err => {
-        console.error(err);
+      models.Individuals.destroy({
+        where: {
+          IndividualID: IndividualID
```

Comment:
If you are writing properties inside an object (between curly brackets) and both the property and the value have the same name like this case then you can just include the property name and javascript would understand it

this line instead of
```javascript
{IndividualID: IndividualID}
```

would be
```javascript
{IndividualID}
```

Again this is for the future when we refactor after the MVP